### ISSUE TITLE
Error should be disabling key not adding user to blackhole

### ISSUE THREAD

**Issue by tommi-gangverk** [jchrisfarris/aws-account-automation#4]
https://github.com/jchrisfarris/aws-account-automation/blob/67eab1d62435cd73dc752b72e821f7218dd6ec80/lambda/ExpireUsers.py#L231

**Comment by caboteria** [comment]
From my read of the code, this happens only if the attempt to deactivate the key fails.

**Comment by tommi-gangverk** [comment]
Yes, the error message is still weird, since there is an error deactivating a key, not sucking someone into a blackhole. :smile:
